components/flow-orchestration-framework/org.wso2.carbon.identity.flow.execution.engine/src/main/java/org/wso2/carbon/identity/flow/execution/engine/FlowExecutionService.java

```diff
@@ -30,8 +31,8 @@
 import org.wso2.carbon.identity.flow.execution.engine.util.FlowExecutionEngineUtils;
 
 import java.util.Map;
-import java.util.Optional;
 
+import static org.wso2.carbon.identity.application.authentication.framework.util.FrameworkUtils.REGISTRATION_FLOW_TYPE;
```

Comment:
Why do we need to define this in framework utils and import here ?
This flow ochestration framework is a separate component, define the constant in that context. And IMO, I think this doesn't even need to be exposed as of now

Reply:
Fixed 6665e49c8edaf17cf5fc9d62f56280752b123777